components/scream/cime_config/buildnml

```diff
@@ -215,6 +229,25 @@ def buildnml(case, caseroot, compname):
 
             fd.write("/\n")
 
+    # Create input data list
+    input_data_list_file = "{}/Buildconf/scream.input_data_list".format(caseroot)
+    if os.path.exists(input_data_list_file):
+        os.remove(input_data_list_file)
+
+    input_file = scream_input["Atmosphere Driver"]["Initial Conditions"]["Physics GLL"]["Filename"]
```

Comment:
Sort of unrelated: this is an ugly implementation detail, which I hope to get rid of. Now that we can put both dynamics and physics fields on the same nc file, the initial conditions should be all in the same file, regardless of the grids where fields live. So I hope to soon rearrange the IC section as
```
  Initial Conditions:
    Filename: blah.nc
```
which has the advantage of not requiring buildnml to know what the grids names are, though I suspect we could infer the physics grid name (like "physics GLL" vs "PG2") from the CASE object.

Reply:
That's fine. It will just require a pretty easy change to buildnml when you change this.